src/App.js

```diff
@@ -39,10 +40,50 @@ const App = ({ appState, world, dialog }) => {
         showFooter = false;
     }
 
+    if (sideMenu) {
+        return (
+            <>
+                <div className={`centered flex-row`}>
+                    <JournalSide disabled={disableJournal} />
+                    <div
+                        className={`centered ${
+                            sideMenu ? 'flex-row' : 'flex-column'
+                        }`}
+                    >
+                        <div className={'centered flex-row'}>
+                            <Viewport>
+                                <World />
+                                <DialogManager />
+                                <Spellbook />
+
+                                {/* Show the floor counter when playing endless mode */}
+                                {gameMode === 'endless' && (
+                                    <EndlessFloorCounter floor={floorNum} />
+                                )}
+                            </Viewport>
+                        </div>
+
+                        <GameMenus />
+                    </div>
+                </div>
+                {showFooter && <Footer />}
+            </>
+        );
+    }
+
     return (
         <>
-            <div className={`centered flex-row`}>
+            <div style={{ float: 'left' }}>
```

Comment:
This hugs the left side of the screen a little, but if you add `marginLeft: '3rem'` it makes it look a *little* nicer (though certainly not perfect).

Reply:
And makes it possible for the game screen and the journal to overlap.